src/main/coordinator.c

```diff
@@ -309,6 +309,6 @@ clear_cdata_counts(cdata_t* cdata)
 	cdata->udf_timeout_count = 0;
 	cdata->udf_error_count = 0;
 
-	as_fence_memory();
+	as_fence_seq();
```

Comment:
I don't think this is needed because the comment states
```
 * note: this is not thread safe, only call this when all other threads are
 * halted
```

Reply:
These are just updating old fences to use the new common fences. I'd rather leave them for now.

Reply:
makes sense